src/__test__/graphql-flow.test.js

```diff
@@ -342,6 +342,67 @@ describe('graphql-flow generation', () => {
             `);
         });
 
+        it('should handle an inline fragment on an interface without a typeCondition', () => {
+            const result = rawQueryToFlowTypes(
+                `
+                query SomeQuery {
+                    hero(episode: JEDI) {
+                        id
+                        ... {
+                            name
+                        }
+                    }
+                }`,
+                {readOnlyArray: false},
+            );
+            expect(result).toMatchInlineSnapshot(`
+                export type SomeQueryType = {|
+                    variables: {||},
+                    response: {|
+                  hero: ?({|
+                    id: string,
+
+                    /** The person's name*/
+                    name: ?string,
+                  |} | {|
+                    id: string,
+                    name: ?string,
+                  |})
+                |}
+                |};
+            `);
+        });
+
+        it('should handle an inline fragment on an object (not an interface)', () => {
+            const result = rawQueryToFlowTypes(
+                `
+                query SomeQuery {
+                    human(id: "hi") {
+                        id
+                        ... {
+                            name
+                        }
+                    }
+                }`,
+                {readOnlyArray: false},
+            );
+            expect(result).toMatchInlineSnapshot(`
+                export type SomeQueryType = {|
+                    variables: {||},
+                    response: {|
+
+                  /** A human character*/
+                  human: ?{|
+                    id: string,
+
+                    /** The person's name*/
+                    name: ?string,
+                  |}
+                |}
+                |};
```

Comment:
super nit: the indentation is a bit off.

Reply:
yeah that's jest's snapshot trying to magic the indentation and not doing an awesome job 🤷‍♂️